src/Mcfedr/PeriodicQueueDriverBundle/Worker/PeriodicWorker.php

```diff
@@ -41,11 +45,15 @@ public function execute(array $arguments)
             throw new UnrecoverableJobException('Missing arguments for periodic job');
         }
 
-        $job = new PeriodicJob($arguments['name'], $arguments['arguments']);
+        $job = new PeriodicJob($arguments['name'], $arguments['arguments'], $arguments['job_tokens']);
```

Comment:
We need to add a bit of backwards compatibility here.

We can check if isset $arguments.job_tokens and if not generate new ones.